### ISSUE TITLE
Crafting pipes don't want to pull from TE finished slots.

### ISSUE THREAD

**Issue by HaniiPuppy** [RS485/LogisticsPipes#546]
Using The Dark Trilogy modpack (1.0.5), with LP 0.8.2.77, Thermal Foundation 1.0.0-RC1-13 and Thermal Expansion 4.0.0-B8-23.

Trying to set up on-request auto-crafting in my LP sorting system, I've found that the crafting pipe placed on the input of a Thermal Expansion machine won't seem to recognise the finished/output slot of the machine and won't pull items out once they're finished.

Placing the crafting pipe on an output side and using sneaky-crafting to insert items into an input side, or placing the crafting pipe on an output side and placing items in the input side using a satellite pipe, results in finished items getting pushed out of the machine and into the LP network, but not fulfilling requests, since it's not the crafting pipe that's pulled the item out, but the block that's pushed the item into the pipe.

For machines with only one input, I'm having the machine output the result into a TE cache and having the crafting pipe next to the cache, inserting ingredients using a satellite pipe, but this work-around obviously doesn't work (or, I've yet to find a similar work-around that does work) for machines that take multiple inputs.

**Comment by bziemons** [comment]
Try the crafting pipe on the input side and set up an output for the machine on an unused side.

**Comment by AartBluestoke** [comment]
More specifically, the crafter pipe checks all sides of the block it is
attached to for output, so for many machines only needs to be out on the
only side, and it will find the output automatically.

**Comment by HaniiPuppy** [comment]
I just tried that, and it works perfectly. Well that's a stupidly easy solution for a problem I was agonising over so much o#o I'll add that to the two wikis so someone else that comes across the same problem see it. Also, I've found doing the sneaky thing I talked about earlier does work fine, if you remove the integrated servo mechanism from the TE machine. Ty ^^

EDIT: Nvm, I can't seem to edit the wikis.
